### DISCUSSION TITLE
Dell U2711-1

### DISCUSSION THREAD

**Discussion by cyberoldie** [xanderfrangos/twinkle-tray#762] [Monitor Compatibility Issues]
I have two of these monitors attached to my NUC computer
According to the Device Manager, the computer has Intel HG Graphics and Radeon RX Vega M GL Graphics support.
The monitors perform completely in all respects - both before and after installing your Twinkle Tray software.  Unfortunately your software only communicates with the first monitor and not the second.  As someone with visual issues I  could really use your software to help me, and would gladly make a contribution if we can get this to work for both monitors

**Comment by Fartom67** [comment]
I have this exact monitor and have tried in vain to control it via Windows with my new notebook via HDMI to HDMI. HDMI to DisplayPort didn't work either. Now I've had success with a new USB C to DisplayPort cable! So it could be that this is your problem with the monitor not working.
